agent/tests/test_php_txn.c

```diff
@@ -8,9 +8,9 @@
 #include "php_agent.h"
 #include "php_txn_private.h"
 
-#define LIBRARY_NAME "php/php-package"
-#define PACKAGE_VERSION "1.2.3"
-#define PACKAGE_MAJOR_VERSION "1"
+#define LIBRARY_NAME "vendor_name/package_name"
+#define LIBRARY_VERSION "1.2.3"
+#define LIBRARY_MAJOR_VERSION "1"
 #define COMPOSER_PACKAGE_VERSION "2.1.3"
```

Comment:
since we know composer version can give versions with letters in them, can we have a case with that?
something like `4.0.x-dev`

Reply:
Since it is presented as a string I don't know anything cares what ASCII values the string is composed of?

Reply:
do we not extract the major version from it? isn't there some string manipulation/scraping at that point?